lib/compare_task.rb

```diff
@@ -41,7 +41,7 @@ def render_html_comparison(description1, description2, scopes, options)
       end
     end
 
-    target = "/tmp/machinery-html-comparison"
+    target = File.join(Machinery::DEFAULT_CONFIG_DIR, "machinery-html-comparison")
```

Comment:
As this is a Machinery specific directory, it would probably be enough to just call it `html-comparison`.

The constant name `DEFAULT_CONFIG_DIR` is misleading, as this is not the directory for configuration, but for all Machinery data. I think we should change that. It makes this patch a bit confusing to read, but if we change it we should do it in a separate patch.